.github/workflows/release-multichannelchat.yml

```diff
@@ -94,21 +88,15 @@ jobs:
           echo "actor-name=$(cargo metadata --no-deps --format-version 1 | jq -r '.packages[].name')" >> $GITHUB_ENV
         working-directory: ${{ matrix.working-directory }}
       - name: Determine actor version
-        if: startswith(github.ref, 'refs/tags/') # Only run on tag push
         run: |
           echo "actor-version=$(cargo metadata --no-deps --format-version 1 | jq -r '.packages[].version')" >> $GITHUB_ENV
         working-directory: ${{ matrix.working-directory }}
-      - name: Determine actor version (main)
-        if: ${{ !startswith(github.ref, 'refs/tags/') }}
-        run: |
-          echo "actor-version=latest" >> $GITHUB_ENV
-        working-directory: ${{ matrix.working-directory }}
       - name: Push actor to AzureCR
         uses: wasmcloud/common-actions/oci-artifact-release@main
         with:
           artifact-path: ${{ matrix.working-directory }}/build/${{ env.actor-name }}_s.wasm
-          oci-url: ${{ secrets.AZURECR_PUSH_URL }}   
-          oci-repository: ${{ env.actor-name }}
+          oci-url: ${{ secrets.AZURECR_PUSH_URL }}
+          oci-repository: mcc_${{ env.actor-name }}
```

Comment:
this is to differentiate between the cosmonic apigateway and the multi-channel chat (mcc) apigateway in `cosmonic.azurecr.io`